### PR TITLE
[MERGED] Dark mode add

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**
image

### CODE REVIEW

**Comment by kuanyui** [comment]
- Missing a border like light one:
Screenshot_20240307_003108
- The background of "currently active engine" is too bright, should also be adjusted to fit dark theme. Maybe `#666666` I guess.
- Remember `:hover` style

File: src/content.ts [hunk 1]
```diff
@@ -207,6 +207,19 @@ async function setupFloatBar() {
...
+    @media(prefers-color-scheme: dark) {
+        #engineSwitcherBar {
+            --bg: #000;
+            border-color: #111;
+        }
+        #engineSwitcherBar img[src$='wikipedia.svg']:not(:hover) {
+            filter: invert(100%);
+        }
+        #engineSwitcherBar a.active {
+            display: none;
```
Comment (kuanyui) [inline src/content.ts]: Why `display:none`??? Totally hide the active one? This cause different behavior between light/dark

Comment (vitaly-zdanevich) [reply src/content.ts]: Maybe in light and dark we will now show the active button? Because when I am on Google - I know it... Or maybe add such option?

**Comment by vitaly-zdanevich** [comment]
Current state:
image

Hover is not changed, because I think it is ok.

**Comment by kuanyui** [comment]
Border is too dim and nearly invisible. It should be brighter, maybe `#777777`

File: src/content.ts (hunk 1 above)
Comment (kuanyui) [reply src/content.ts]: Yes, I think you can add an option if you want such behavior. Because I personally still prefer the original behavior.

**Comment by vitaly-zdanevich** [comment]
With #333 its absolutely visible
image

**Comment by kuanyui** [comment]
Squashed and merged. And also refactored with CSS variables instead of duplicated rules definitions, see: https://github.com/kuanyui/EngineSwitcher/commit/5c6ab432e43b045f81858043f8aadf05857afb87

Thanks for your contribution!

**Pull request closed without merging**